sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -206,6 +213,15 @@ func runCtr(containerdSocket string, namespace string, containerID string, sourc
 
 	// If the container doesn't already exist, create it
 	if container == nil {
+		// Set the destination name for the container persistent storage location
+		var persistentPath string
```

Comment:
Nit: I think this is a directory name, not a complete path.

```suggestion
		var persistentDir string
```

Reply:
I refactored the code a bit, and I used your suggestion. Let me know if you have any concern with the new approach I used.